homeassistant/components/withings/sensor.py

```diff
@@ -531,15 +530,13 @@ class WithingsActivitySensorEntityDescription(SensorEntityDescription):
         value_fn=lambda activity: activity.active_calories_burnt,
         suggested_display_precision=1,
         translation_key="activity_active_calories_burnt_today",
-        native_unit_of_measurement="calories",
```

Comment:
Oh for the calories i still need to switch to the energy device class since energy supports calories now

Reply:
Oh, I'll revisit this one sometime later then.